src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 7) {
+    return (days * 40 - 50);
```

Comment:
brackets are not required here, remove them.

Reply:
Changes were made.